polkadot-parachains/parachains-common/src/lib.rs

```diff
@@ -17,8 +17,15 @@
 
 pub mod impls;
 pub use constants::*;
+use core::marker::PhantomData;
+use frame_support::weights::Weight;
 pub use opaque::*;
 pub use types::*;
+use xcm::{
+	opaque::v2::prelude::{DepositReserveAsset, TransferReserveAsset},
```

Comment:
Just import `xcm::latest::prelude::*` instead of importing types one by one.

Reply:
all hell breaks loose if I do that. I think it upsets the other imports in the file. Maybe I should break it out into its own file...